content/pages/meetup/2023/0920-nycu.rst

```diff
@@ -0,0 +1,105 @@
+========================================
+Meetup 2023 September 20th in NYCU
+========================================
+
+:date: 2023-08-31 13:00
+:url: meetup/2023/0920-nycu
+:save_as: meetup/2023/0920-nycu.html
+
+(DRAFT)
+
+Date
+-----
+
+* Date: September 20th, 2023
+* Time: 18:30 -- 21:30 (3 hours)
+
+|
+
+Agenda
+--------
+
```

Comment:
Please add a time entry for check-in.  We may use 20-30 minutes for checking in and icebreaking.